control-plane/roles/gardener-projects/tasks/project.yaml

```diff
@@ -0,0 +1,6 @@
+---
+- name: Create Gardener project
+  k8s:
+    definition: "{{ lookup('template', 'project.yaml.j2') }}"
+    kubeconfig: "{{ virtual_garden_kubeconfig }}"
```

Comment:
Wrong namespace.
```suggestion
    kubeconfig: "{{ gardener_projects_virtual_garden_kubeconfig }}"
```

Reply:
This an ansible fact returned by the `virtual_garden_kubeconfig` module.